services/apps/portal/src/document/document.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { Base } from '../common/entities/base.entity';
+import { User } from '../user/user.entity';
+
+@Entity()
+export class Document extends Base {
+  constructor(data?: Partial<Document>) {
```

Comment:
Did you delete mimeType on purpose?

Reply:
mimeType was on the DTO but not the entity, so it never existed in our DB and always mapped to null.